### ISSUE TITLE
Text elements must have sufficient color contrast against the background

### ISSUE THREAD

**Issue by GGD4V** [MeD-DMC/adapt_authoring_college#4]
https://esdc.prv/cgi-bin/scdtic-ictdcs/page-error-view.aspx?id=14884&wid=0

**Comment by petchen123** [comment]
Solution for the color-contrast issue on the navigation bar is to change the color in css for `.navitation-item` to black 

![image](https://github.com/MeD-DMC/adapt_authoring_college/assets/25251968/f124c090-1d38-4fa6-82a5-dd3d87f3a549)


![image](https://github.com/MeD-DMC/adapt_authoring_college/assets/25251968/f2713000-1f82-4cc6-ba92-dcff5f0aa635)

**Comment by petchen123** [comment]
`.project-list-item` also needs to have black color

![image](https://github.com/MeD-DMC/adapt_authoring_college/assets/25251968/d629ea17-1e7a-4b6a-a135-f31083db7489)

**Comment by petchen123** [comment]
`.action-primary` also needs to be black

![image](https://github.com/MeD-DMC/adapt_authoring_college/assets/25251968/16ecea37-a7b4-436f-9a5c-451944ebd9a4)

**Comment by petchen123** [comment]
`.sidebar .sidebar-row button, .sidebar .sidebar-breadcrumb button, .sidebar .sidebar-row a, .sidebar .sidebar-breadcrumb a` needs to be white

![image](https://github.com/MeD-DMC/adapt_authoring_college/assets/25251968/0abe04be-16d9-42a6-9a55-aa6d52a796fa)

**Comment by petchen123** [comment]
`.action-secondary` needs to be white

![image](https://github.com/MeD-DMC/adapt_authoring_college/assets/25251968/c5ec11e7-ff6d-475e-b841-c8db0821c97f)
